tests/test_regression/data/losses/adversarialbcewithlogits.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
maybe it would be better to have these encoded as structured information instead of the string that represents the class? then in the tests, it could be instantiated?

I would also say we could encode them all in a single JSON rather than many small ones

Reply:
Sounds good to me. 👍